doc/sphinx/addendum/type-classes.rst

```diff
@@ -317,13 +317,21 @@ Command summary
    :attr:`universes(polymorphic)`, :attr:`universes(template)`,
    :attr:`universes(cumulative)` and :attr:`private(matching)` attributes.
 
+   It also supports the :attr:`mode` attribute for setting a hint mode
+   declaration for the class.
+
    .. note::
       Don't confuse typeclasses with "coercion classes", described in
       `implicit coercions<classes-implicit-coercions>`.
 
    When record syntax is used, this command also supports the
    :attr:`projections(primitive)` :term:`attribute`.
 
+   .. attr:: mode({+ {| + | ! | - } })
+      :name: mode
+
+      Sets the mode of resolution for queries on the class. The syntax is explained in :cmd:`Hint Mode`.
```

Comment:
The syntax seems wrong here, e.g. it is `mode="+ + - !"` not `mode=(+ + - !)`

Reply:
Indeed, I did the same as is done for the warning attribute and refered to Hint Mode for the syntax.